packages/components/date-input/src/date-input-group.tsx

```diff
@@ -64,7 +64,9 @@ export const DateInputGroup = forwardRef<"div", DateInputGroupProps>((props, ref
         {isInvalid && errorMessage ? (
           <div {...errorMessageProps}>{errorMessage}</div>
         ) : description ? (
-          <div {...descriptionProps}>{description}</div>
+          <div {...descriptionProps}>
+            {description === " " ? <span>&#8203;</span> : description}
```

Comment:
1. It's not immediately clear for users that `" "` causes a span to appear where normally you'd expect text
2. Doesn't seem ideal for accessibility

Reply:
Also this won't work if neighboring input have multiple lines of description.